lib/adapters/neo4j.js

```diff
@@ -0,0 +1,426 @@
+var RSVP = require('rsvp');
+var _ = require('lodash');
+var moment = require("moment");
+var Promise = RSVP.Promise;
+var adapter = {};
+var neo4j = require('neo4j');
+var mongoose = require('mongoose');
+
+adapter._init = function (options) {
+  // var connectionString = options.connectionString;
+
+  // if (!connectionString || !connectionString.length) {
+  //   connectionString = 'mongodb://' +
+  //     (options.username ? options.username + ':' + options.password + '@' : '') +
+  //     options.host + (options.port ? ':' + options.port : '') + '/' + options.db;
+  // }
+
+  // mongoose.set('debug', options.debug);
+
+  //Setup mongoose instance
+  this.db = new neo4j.GraphDatabase(options.connectionString);
+  
+  console.log("Init DB", this.db);
+};
+
+/**
+ * Store models in an object here.
+ *
+ * @api private
+ */
+adapter._models = {};
+_schema = {};
+
+adapter.schema = function (name, schema, options, schemaCallback) {
```

Comment:
This function should extend / modify schema for some specific adapter needs. If anything here is not specific to adapter i believe we should just move this to `fortune.resource()`. For instance this `//Convert strings to associations` seems to be generic enough to belong to fortune.